test/unittest/main.cpp

```diff
@@ -0,0 +1,16 @@
+#include <cstdlib>
+
+#include <gmock/gmock.h>
+#include <gtest/gtest.h>
+
+#include <fcntl.h>
+#include <unistd.h>
+
+int main(int argc, char *argv[]) {
+  int seed = 12345;
+  srand(seed);
+  /* set_seed(); */
```

Comment:
I used to set seed from /dev/urandom, to run tests in a loop, but this behaviour was changed.This is a leftover which will be removed.